include/openssl/store.h

```diff
@@ -57,6 +57,11 @@ OSSL_STORE_CTX *OSSL_STORE_open(const char *uri, const UI_METHOD *ui_method,
                                 OSSL_STORE_post_process_info_fn post_process,
                                 void *post_process_data);
 
+OSSL_STORE_CTX *OSSL_STORE_open_with_libctx(
+    const char *uri, const UI_METHOD *ui_method, void *ui_data,
+    OSSL_STORE_post_process_info_fn post_process, void *post_process_data,
+    OPENSSL_CTX *libctx, const char *propq);
```

Comment:
Source code style wise, I believe this isn't OK, as parenthesised things should be aligned with the left parenthesis.

However, I see nothing wrong with this, and I believe it fits our coding style:

``` C
OSSL_STORE_CTX *OSSL_STORE_open_with_libctx
    (const char *uri, const UI_METHOD *ui_method, void *ui_data,
     OSSL_STORE_post_process_info_fn post_process, void *post_process_data,
     OPENSSL_CTX *libctx, const char *propq);
```

Reply:
Err.. this style is done all over the place and is (arguably) more readable than the proposed alternative.

Just as this is more readable (also arguably)
```
if (....) {
}
```

Reply:
@levitte are you going to block on either of the above two comments?

Reply:
I have no idea how you think a function declaration compares to an if statement.  They are completely different in my mind (apart from using parentheses in their respective syntax)

Reply:
I feel strongly about the \< name, libctx, propq \> tuple staying together.  The code style issue, less so, but I *am* worried that we stray more and more from the style guide, and re-create the mess we had before.

(the coding style needs an update, and there's been several proposals, but not final decisions...)

Reply:
> I have no idea how you think a function declaration compares to an if statement.

As far as I am aware the reason for this format
```
if () {
}
```
Is related to scan-ability of the line.
So I am not sure why you think this can't also apply to () when it needs to span multiple lines.

Reply:
I have complied in this PR.. It doesn't mean I have to agree :)